### ISSUE TITLE
Allow publicizing an entire type

### ISSUE THREAD

**Issue by gotmachine** [krafs/Publicizer#100]
A common use case when publicizing the whole assembly isn't desirable is to publicize all members in a specific type (and the type itself).
Currently, this is quite cumbersome, as it requires to publicize the type, and every member one by one.

Would it be possible to have such an option ?

For example, something like :
```xml
<Publicize Include="MyAssemblyFileName:MyNamespace.MyType" PublicizeMembers="true" />
```

**Comment by krafs** [comment]
I think this is a good idea!

My first thought is to maybe change the name of the attribute to be more in line with existing attributes. 
We can already specify these attributes:
```xml
<Publicize Include="ExampleAssembly" IncludeCompilerGeneratedMembers="false" IncludeVirtualMembers="false" />
```

So the new attribute is perhaps better named e.g. `IncludeMembers`.

Then there's the confusing bit about the existing attributes only being applicable when specifying a whole assembly. If specifying e.g. a type they don't actually do anything. So introducing e.g. an `IncludeMembers` attribute that only applies when specifying a type could get confusing. But maybe there's some way this can be unified somehow.

It's also not clear what people expect "Members" to mean. If I want to publicize a Type, including all its members, do I expect nested types and their members in turn to be publicized, or just the immediate fields, properties and methods of the specified type?

**Comment by ZzZombo** [comment]
I want to voice my support for this!

**Comment by RedMattis** [comment]
Ditto.

It would be very nice feature. I often see stuff like "static class BlarghUtilities" with a lot of useful private static methods/properties/fields, and I feel exposing dozens of them individually could make the project file a bit bloaty.

**Comment by krafs** [comment]
Added in #123 . Now available in [v2.3.0](https://github.com/krafs/Publicizer/releases/tag/v2.3.0).

`Publicize`-items with just the assembly in the `Include` can now optionally have an attribute or node called `MemberPattern`. `MemberPattern` accepts a regular expression that will be matched against the members in the assembly.

This way all members in a type can be selected in one go, like this:
```xml
<Publicize Include="MyAssembly" MemberPattern="^MyNamespace\.MyType\..*" />
```

Please let me know if anything seems unclear!
